src/components/FlawLabels/FlawLabelsTable.vue

```diff
@@ -31,7 +31,9 @@ const availableLabels = computed(() =>
     !Object.values(labels.value).some(({ label }) => label === contextLabel),
   ),
 );
-const [isExpanded, toggleExpanded] = useToggle(true);
+
+const isExpandedDefault = labelsFromProps.value.some(label => label.contributor) || ([...newLabels.value].length > 0);
```

Comment:
The `([...newLabels.value].length > 0)` assertion is not working as expected, cause `newLabels` is only for labels just added and not saved in OSIDB, not for actual labels with "NEW" state.

Something like this should work:
```suggestion
const isExpandedDefault = labelsFromProps.value.some(label => label.contributor || label.state === 'NEW');
```

Reply:
Thank you!